### PR TITLE
Update grpc-gateway dependency

### PR DESCRIPTION
**Pull request by johanbrandhorst**
This fixes an issue which would allow an attacker to trigger a
(recovered) panic in the gateway handler.

### CODE REVIEW

File: internal/gen/controller.swagger.json [hunk 1]
```diff
@@ -3729,6 +3729,33 @@
         ]
       }
     },
+    "/v1/workers/:create:worker-led": {
```
Comment (johanbrandhorst) [inline internal/gen/controller.swagger.json:3732]: This path changed 🤔

Comment (johanbrandhorst) [reply internal/gen/controller.swagger.json:3732]: This change appears to be limited to the openapiv2 generator, the handler is still doing the same thing.

Comment (johanbrandhorst) [reply internal/gen/controller.swagger.json:3732]: I'm pretty sure this is caused by https://github.com/grpc-ecosystem/grpc-gateway/pull/2727

Comment (jefferai) [reply internal/gen/controller.swagger.json:3732]: Can we wait to merge this until that PR is pulled in too?

File: sdk/pbs/controller/api/resources/groups/group.pb.go [hunk 2]
```diff
@@ -1,6 +1,6 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.27.1
+// 	protoc-gen-go v1.28.1
```
Comment (jefferai) [inline sdk/pbs/controller/api/resources/groups/group.pb.go:3]: Can you update the tools file if we're going to standardize on this for the next release?

**Review by jefferai — CHANGES_REQUESTED** [review]

File: internal/gen/controller.swagger.json (hunk 1 above)
Comment (johanbrandhorst) [reply internal/gen/controller.swagger.json:3732]: I've got a fix in https://github.com/grpc-ecosystem/grpc-gateway/pull/2825.

File: sdk/pbs/controller/api/resources/groups/group.pb.go (hunk 2 above)
Comment (johanbrandhorst) [reply sdk/pbs/controller/api/resources/groups/group.pb.go:3]: The tools file doesn't contain versions, only modules, right? The version is already controlled by `go.mod`, which is updated by this PR.

**Comment by johanbrandhorst** [comment]
I don't think we need to get this in for 0.10.0. The impact of this bug is that an attacker can basically spam the logs of the controller with panic messages.

File: internal/gen/controller.swagger.json (hunk 1 above)
Comment (johanbrandhorst) [reply internal/gen/controller.swagger.json]: I can confirm that this PR fixes the issue for us. I would like to wait for one of the other gateway contributors to comment on it before merging though.

**Comment by johanbrandhorst** [comment]
This is ready for review again, we can merge once 0.10.0 is out.

**Pull request merged**